eks_rolling_update.py

```diff
@@ -179,13 +179,16 @@ def update_asgs(asgs, cluster_name):
             modify_aws_autoscaling(asg_name, "suspend")
 
         # start draining and terminating
+        desired_asg_capacity = asg_state_dict[asg_name][0]
         for outdated in outdated_instances:
             # catch any failures so we can resume aws autoscaling
             try:
                 # get the k8s node name instead of instance id
                 node_name = get_node_by_instance_id(k8s_nodes, outdated['InstanceId'])
+                desired_asg_capacity -= 1
```

Comment:
Is this subtracting 1 from desired capacity?

Reply:
being done before you terminate/delete the node. This will also update the tag